backend/eduschola_project/edusch_app/views.py

```diff
@@ -57,83 +61,96 @@ class ParentListCreateView(generics.ListCreateAPIView):
     def createParent(self, request, *args, **kwargs):
```

Comment:
There seems to be a bit of confusion in the naming of the method. Your view is named ParentListCreateView, but the create method is named createParent which doesn't override the default create method provided by the ListCreateAPIView. This would mean that your createParent method isn't being called when a POST request is made to the view